WalletWasabi/Wallets/IWallet.cs

```diff
@@ -25,7 +25,7 @@ public interface IWallet
 
 	Task<bool> IsWalletPrivateAsync();
 
-	Task<IEnumerable<SmartCoin>> GetCoinjoinCoinCandidatesAsync();
+	Task<CoinsRegistry> GetCoinjoinCoinCandidatesAsync();
```

Comment:
This gonna break the affiliation integrations. Is this really necessary?

Reply:
It is, the `GetCoinjoinCoinCandidatesAsync` needs to give back a CoinRegistry here so we can call the `AvailableForCoinJoin` function,
https://github.com/zkSNACKs/WalletWasabi/blob/280e4c60ce8351433df0a4f112b9376ebb971b14/WalletWasabi/WabiSabi/Client/CoinJoinManager.cs#L579-L587

Reply:
This gonna break the affiliation integrations?